### PR TITLE
Fixing rhn_register upload certificate fail  #114

### PR DESCRIPTION
**Pull request by KrE80r**

### CODE REVIEW

**Comment by budhrg** [comment]
LGTM :+1: as `tmp` is never defined before the changed line.

@KrE80r Thanks for the PR.

File: plugins/guests/redhat/cap/rhn_register.rb [hunk 1]
```diff
@@ -81,7 +81,7 @@ def self.rhn_register_upload_certificate(machine, ui)
             if File.exist?(machine.config.registration.ca_cert)
               # Make sure the provided CA certificate file will be configured
               cert_file_name = File.basename(machine.config.registration.ca_cert)
-              cert_file_content = File.read(machine.config.registration.ca_cert, tmp)
+              cert_file_content = File.read(machine.config.registration.ca_cert)
```
Comment (LalatenduMohanty) [inline plugins/guests/redhat/cap/rhn_register.rb]: @KrE80r Any idea why the `tmp` is in here? I am trying to understand the root cause of the regression. cc @budhrg

**Comment by LalatenduMohanty** [comment]
Looks like it was a mistake with code rebase. Check the below commits. `43550d50b7557814df393a50c37f0114048791d6` fixes the issue and then the next commit `3de3773498e0c7216c347076610594be6699763e` removes the fix.

```
commit 3de3773498e0c7216c347076610594be6699763e
Author: Horatiu Eugen Vlad <horatiu@vlad.eu>
Date:   Tue Jan 5 15:56:58 2016 +0100

    Use machine.communicate.sudo method instead of execute

commit 43550d50b7557814df393a50c37f0114048791d6
Author: Horatiu Eugen Vlad <horatiu@vlad.eu>
Date:   Tue Jan 5 16:23:00 2016 +0100

    Fixes #60: rhn_register upload certificate fails
```

**Comment by LalatenduMohanty** [comment]
LGTM.

**Comment by LalatenduMohanty** [comment]
Merged

**Pull request closed without merging**